Userland/Demos/Mandelbrot/Mandelbrot.cpp

```diff
@@ -392,13 +408,33 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto mandelbrot = TRY(window->try_set_main_widget<Mandelbrot>());
 
     auto file_menu = TRY(window->try_add_menu("&File"));
-    TRY(file_menu->try_add_action(GUI::Action::create("&Export...", { Mod_Ctrl | Mod_Shift, Key_S }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/save.png").release_value_but_fixme_should_propagate_errors(),
+
+    auto& export_submenu = file_menu->add_submenu("&Export");
+
+    TRY(export_submenu.try_add_action(GUI::Action::create("As &BMP",
+        [&](GUI::Action&) {
+            Optional<String> export_path = GUI::FilePicker::get_save_filepath(window, "untitled", "bmp");
+            if (!export_path.has_value())
+                return;
+            mandelbrot->export_image(export_path.value(), ImageType::BMP);
+        })));
+    TRY(export_submenu.try_add_action(GUI::Action::create("As &PNG", { Mod_Ctrl | Mod_Shift, Key_S },
         [&](GUI::Action&) {
             Optional<String> export_path = GUI::FilePicker::get_save_filepath(window, "untitled", "png");
             if (!export_path.has_value())
                 return;
-            mandelbrot->export_image(export_path.value());
+            mandelbrot->export_image(export_path.value(), ImageType::PNG);
         })));
+    TRY(export_submenu.try_add_action(GUI::Action::create("As &QOI",
+        [&](GUI::Action&) {
+            Optional<String> export_path = GUI::FilePicker::get_save_filepath(window, "untitled", "qoi");
+            if (!export_path.has_value())
+                return;
+            mandelbrot->export_image(export_path.value(), ImageType::QOI);
+        })));
+
+    export_submenu.set_icon(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/save.png").release_value_but_fixme_should_propagate_errors());
```

Comment:
Looks like we don't need to `release_value_but_fixme_should_propagate_errors` here?
```c++
export_submenu.set_icon(TRY(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/save.png")));
```